.yarnrc

```diff
@@ -0,0 +1,6 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
+
+
+yarn-path ".yarn/releases/yarn-1.22.18.cjs"
+--ignore-engines true
```

Comment:
ignore `engines` field for lerna-lite…

Reply:
Blocker: https://github.com/ghiscoding/lerna-lite/issues/77

We will wait for this issue result.